lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -88,20 +89,38 @@ class LighthouseReportViewer {
 
   /**
    * Attempts to pull gist id from URL and render report from it.
-   * @return {Promise<void>}
    * @private
    */
   _loadFromDeepLink() {
     const params = new URLSearchParams(location.search);
+
+    let provider = params.get('provider');
```

Comment:
this also seems like overkill (especially since PSI support is going to be temporary (quoth @paulirish)).

`gist` is shorthand for "load this gist id", seems fine to use `psi` for "run PSI with this url" (though we can obviously pick another search param name, psiurl...)

Reply:
where did @paulirish mention this being temporary? I wouldn't want URLs using this feature to die in the future.

Reply:
`psiurl` sgtm

this functionality should continue to work but the extension will point at web.dev when that's ready and stop linking to the viewer.

Reply:
okay and we'll, at some point, redirect (selectively) from the viewer to web.dev for all these `psiurl`s.